server/models/list.py

```diff
@@ -1,12 +1,32 @@
 from database import db
 
+
 class List(db.Model):
 
     __tablename__ = 'list'
 
-    id = db.Column(db.Integer, primary_key=True,autoincrement=True)
-    product_type = db.Column(db.String, unique=True, nullable=False)
-    # have an image type aswell, will implement later...
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'),nullable=False)
+    name = db.Column(db.String, nullable=False)
+    img_url = db.Column(db.String, nullable=False)
+    product_count = db.Column(db.Integer, nullable=False)
```

Comment:
Do we actually need this field?

Reply:
It will likely be much easier to use`Array.length`on the frontend to count the amount of products. Having a product_count value in the db opens up the possibility of it going out of sync with the true count.